templates/error.html

```diff
@@ -0,0 +1,139 @@
+{% extends 'base.htm' %}
+
+{% block content %}
+    <!-- This template was made by Colorlib (https://colorlib.com)
```

Comment:
Can you remove the Colorlib comments?

Reply:
The template is licensed under [CC BY 3.0](https://creativecommons.org/licenses/by/3.0/). I think we have to give credit somewhere, but it doesn't necessarily have to be in a comment. Or we could [buy the template](https://colorlib.com/wp/licence/) for $29. Or we could get rid of this template and make our own. 

Thoughts?

Reply:
Ah, I see.  Let's keep the attribution for now, then, and when we modify the styles and content sufficiently (to make the error page more cohesive with the rest of the site), we can get rid of it.